packages/esm-patient-vitals-app/src/biometrics/biometrics-base.component.tsx

```diff
@@ -36,28 +37,50 @@ const BiometricsBase: React.FC<BiometricsBaseProps> = ({ patientUuid, pageSize,
     [config, currentVisit],
   );
 
-  const tableHeaders = [
-    { key: 'date', header: t('dateAndTime', 'Date and time'), isSortable: true },
-    { key: 'weight', header: withUnit(t('weight', 'Weight'), conceptUnits.get(config.concepts.weightUuid) ?? '') },
-    { key: 'height', header: withUnit(t('height', 'Height'), conceptUnits.get(config.concepts.heightUuid) ?? '') },
-    { key: 'bmi', header: `${t('bmi', 'BMI')} (${bmiUnit})` },
+  const tableHeaders: Array<BiometricsTableHeader> = [
     {
-      key: 'muac',
+      key: 'dateRender',
```

Comment:
What's the rationale behind the `Render` suffix?

Reply:
There are 2 kinds of data for a column (say `date` column): 
1. `date`, which is the ISO string
2. `dateRender`, which is the formatted version of the pt.1 date.

Now, if we see, the user/ datatable sees the `dateRender` on the screen, but we want to sort the column based on the `date` value, hence I added the "Render" suffix.